.travis.yml

```diff
@@ -1,14 +1,5 @@
 before_install:
-  # Repo for Yarn
```

Comment:
All this stuff isn't needed anymore since it's built into travis now and we are checking in our yarn.lock file.